lib/vagrant/cli.rb

```diff
@@ -30,6 +31,20 @@ def execute
         return 0
       end
 
+      if @main_args.include?("-install-autocomplete")
+        # Install autocomplete scripts
+        written_paths = Vagrant::Util::InstallCLIAutocomplete.install
+        if written_paths.length > 0
+          @env.ui.info("Autocomplete installed at paths:")
+          written_paths.each do |p|
+            @env.ui.info("- #{p}")
```

Comment:
Instead of printing each line by itself, we should build up the string here and print it all at once afterwords. Otherwise the debug output or machine readable output will end up with a bunch of extra stuff in between each line from invoking `@env.ui.info` for each path.